packages/amplify-e2e-tests/src/__tests__/init_d.test.ts

```diff
@@ -32,7 +33,15 @@ describe('amplify init d', () => {
     const localEnvData = fs.readJsonSync(localEnvPath);
     const originalPath = localEnvData.projectPath;
 
-    expect(localEnvData.projectPath).toEqual(fs.realpathSync(projRoot));
+    if (isWindowsPlatform()) {
+      // The project dir created using os.tmpdir() has shortened C:\\Users\\CIRCLE~1.PAC\\ segment
+      // but projectPath comes as C:\\Users\\circleci.PACKER-633B1A5A\\ for example
+      // which is fine but makes comparison harder
+      expect(path.isAbsolute(localEnvData.projectPath)).toBe(true);
+      expect(fs.statSync(localEnvData.projectPath)).toEqual(fs.statSync(projRoot));
+    } else {
+      expect(localEnvData.projectPath).toEqual(fs.realpathSync(projRoot));
+    }
```

Comment:
New image has stuff like `C:\\Users\\circleci.PACKER-633B1A5A` as home directory.
Old image had `C:\\Users\\circleci`. Hence the change here.